lca-chimevc-stack/lambda_functions/chime_call_analytics_initialization/index.js

```diff
@@ -623,11 +626,15 @@ const handler = async function handler(event, context) {
         );
         return;
       }
+      
+      let tempCallData = await getCallDataFromChimeEvents(event);
+      callData = await getCallDataFromDdb(tempCallData.callId);
```

Comment:
why bother building `tempCallData`? I think you just need `event.detail.callId`, right?